great-idea/contact.html

```diff
@@ -0,0 +1,97 @@
+                    <div class="address">123 Way 456 Street<br />Somewhere, USA</div>
+                    <div class="phone">
+                        <img src="/great-idea/img/phone-not-being-used.png" />
+                        <p>(888) 888-8888</p>
+                    </div>
+                    <div class="email">
+                        <img src="/great-idea/img/icons8-envelope-100.png" />
+                        <p>sales@greatidea.io</p>
+                    </div>
+                    <div class="twitter">
+                        <img src="/great-idea/img/twitter.png" />
+                        <p>@GREATIDEA</p>
+                    </div>
+                    <div class="facebook">
+                        <img src="/great-idea/img/facebook.png" />
+                        <p>/GREATIDEA</p>
+                    </div>
+                    <div class="instagram">
+                        <img src="/great-idea/img/instagram-new.png" />
+                        <p>/GREATIDEA</p>
+                    </div>
+                </div>
+                <div class="contact-right">
+                        <div class="contact-header">
+                            <p class="contact-header-left">02 /</p>
+                            <p class="contact-header-right">SEND US<br />A MESSAGE</p>
+                        </div>
+                        <p>Your name <span class="required">(required)</span></p>
+                        <input type="text" name="name" size="50" maxlength="50" />
+
+                        <p>Email address <span class="required">(required)</span></p>
+                        <input type="text" name="email" size="50" maxlength="50" />
+
+                        <p>Telephone number</p>
+                        <input type="text" name="telephone-number" size="50" maxlength="20" />
+
+                        <p>Your message <span class="required">(required)</span></p>
+                        <textarea name="message" cols="50" rows="10"></textarea>
+
+                        <p class="privacy-policy">Please read our <a href="#">Privacy Policy</a> to find out how we use your information.</p>
+
+                        <div class="terms-agreement">
+                            <input type="checkbox" name="terms-agreement" value="terms-agreement" /><p>I have read, understood and agree to the terms outlined in the Privacy Policy</p>
+                        </div>
+
+                        <button class="send-message">Send Message</button>
+
+                </div>
+            </div>
+            <div class="map">
+                <iframe src="https://www.google.com/maps/embed?pb=!1m16!1m12!1m3!1d26373872.52705101!2d-113.7657689655779!3d36.203845851326214!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!2m1!1s123+Way+456+Street+Somewhere%2C+USA!5e0!3m2!1sen!2sus!4v1544237427598" width="100%" height="300" frameborder="0" style="border:0" allowfullscreen></iframe>
```

Comment:
awesome use of an iframe to add a google maps section, I would have been happy with just an image for this static project but you went above and beyond. Good job, the best part about stretch goals is that they really let you have fun with the projects.

Reply:
yeap, love stretch goals because they give one more freedom and the possibility to be creative.